services/ui-src/src/measures/2025/CPAAD/questions/DataSource.tsx

```diff
@@ -28,16 +28,7 @@ export const DataSource = ({ type }: DataSourceProps) => {
             value: "Other",
             children: [
               <QMR.TextArea
-                label={
-                  <>
-                    Describe the data source (
-                    <em>
-                      text in this field is included in publicly-reported
-                      state-specific comments
-                    </em>
-                    ):
-                  </>
-                }
+                label={<>Describe the data source:</>}
```

Comment:
Tiny suggestion: We can get away with just a string here.

```suggestion
                label="Describe the data source:"
```